js/src/util.js

```diff
@@ -154,6 +154,10 @@ const Util = (() => {
           }
         }
       }
+    },
+
+    get jQuery() {
+      return window.$ || window.jQuery
```

Comment:
Maybe we should change the order since `jQuery` should always be available?

Reply:
done :wink: